### PR TITLE
SB-46: Add support for translations

### PR DESCRIPTION
**Pull request by raiot**
# What's the problem?
We currently don't support translations.

# What's the solution?
Implement react-i18next to handle translations and replace the text with the labels

NOTE: Translation labels are temporary, team comments are welcome.

### CODE REVIEW

File: src/store/reducers/settings.js [hunk 1]
```diff
@@ -23,15 +25,17 @@ if (!preferences) {
 const settingsReducer = (state = initialState, { type, payload }) => {
   switch (type) {
     case SETTINGS_SET_AUTO_PLAY:
-      setPreferences({ ...state, autoPlay: payload });
+      setPreferences({ ...state.toJS(), autoPlay: payload });
```
Comment (tonymtz) [inline src/store/reducers/settings.js]: ugh... this `.toJS()` doesn't seem right.

**Review by tonymtz — APPROVED** [review]

File: src/store/reducers/settings.js (hunk 1 above)
Comment (raiot) [reply src/store/reducers/settings.js]: Yeah, but I was having problems serializing/deserializing from map into localstorage, I’ll have a second look later to see what can be done

Comment (tonymtz) [reply src/store/reducers/settings.js]: We could also update the localStorage `window.beforeunload`.

File: public/locales/es-MX/translation.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
 {
+    "common": {
+        "privacy": "Politicas de privacidad",
+        "tos": "Términos de servicio"
+    },
+    "landing": {
+        "title": "Songbox",
+        "login": "Inicia Sesión",
+        "message": "Hola! Nos da gusto volverte a ver."
```
Comment (tonymtz) [inline public/locales/es-MX/translation.json]: ¡Hola!

**Pull request merged**